syntax = "proto3";

package util_models;

enum SupplyType {
	AidKit = 0;
	AmmoCrate = 1;
}